server.js

```diff
@@ -49,12 +49,12 @@ app.get('/api', function apiIndex(req, res) {
   res.json({
     woopsIForgotToDocumentAllMyEndpoints: true, // CHANGE ME ;)
```

Comment:
Since you have updated all the below lines, you can change this flag to false. Or probably delete the whole line altogether so it does not show up.